drizzle.config.ts

```
@@ -0,0 +1,20 @@
+import "dotenv/config";
+
+import { defineConfig } from "drizzle-kit";
+
+export default defineConfig({
+  dialect: "postgresql",
+  schema: "./src/db/schema.ts",
+  out: "./drizzle",
+  dbCredentials: {
+    url: (() => {
+      const url = process.env.POSTGRES_URL ?? process.env.POSTGRES_PRISMA_URL;
```

ESLint

[error] Unexpected use of process.env.

(node/no-process-env)